### ISSUE TITLE
The compiler-arg can't be used to pass flags with dashes

### ISSUE THREAD

**Issue by roblabla** [volks73/cargo-wix#108]
```
Z:\project> cargo wix --compiler-arg -ext
error: Found argument '-e' which wasn't expected, or isn't valid in this context

USAGE:
    cargo wix --compiler-arg <compiler-arg>...

For more information try --help
```

I believe this would be solved by setting [`allow_hyphen_values`](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.allow_hyphen_values) on the `compiler-arg` arg. The same probably needs to be done to `linker-arg`.

**Comment by volks73** [comment]
I very quickly tried `cargo wix --compiler-arg "-e"` and I get the same error. I agree the `allow_hyphen_values` setting should be used for both the `compiler-arg` and `linker-arg` CLI arguments.

**Comment by volks73** [comment]
Fixed as of 3e7bb61897639e06b759d9057a81b943abf3d52f.

The `number_of_values(1)` method also needed to be added. In combination with the `allow_hyphen_values` method, this avoid ambiguity in argument parsing. The combination allows for multiple occurrences but only one value per occurrences. The long help and module-level documentation for the binary are updated with this information as well (just not published yet).
